eclipse/src/saros/editor/EditorManager.java

```diff
@@ -633,14 +633,6 @@ void textAboutToBeChanged(int offset, String text, int replaceLength, IDocument
 
     // inform all registered ISharedEditorListeners about this text edit
     editorListenerDispatch.textEdited(textEdit);
-
-    /*
-     * TODO Investigate if this is really needed here
-     */
-    IEditorInput input = changedEditor.getEditorInput();
-    IDocumentProvider provider = EditorAPI.getDocumentProvider(input);
-    IAnnotationModel model = provider.getAnnotationModel(input);
-    contributionAnnotationManager.splitAnnotation(model, offset);
```

Comment:
Can you please have a look at the conversation history ? I already talked about another issue in this file on line 1086.